packages/block-editor/src/components/inserter/menu.js

```diff
@@ -58,6 +58,11 @@ function InserterMenu(
 		( select ) => unlock( select( blockEditorStore ) ).isZoomOut(),
 		[]
 	);
+	const hasSectionRootClientId = useSelect( ( select ) => {
+		const { getSectionRootClientId } = unlock( select( blockEditorStore ) );
+		const theRoot = getSectionRootClientId();
+		return !! theRoot?.length > 0;
```

Comment:
```suggestion
		return theRoot?.length > 0;
```

`theRoot?.length > 0 ` already produces a Boolean result so no need to coerce

Reply:
👍